imports/plugins/included/product-admin/client/components/ProductsTable.js

```diff
@@ -60,6 +71,27 @@ function ProductsTable() {
   const history = useHistory();
   const [shopId] = useCurrentShopId();
   const [createProduct, { error: createProductError }] = useMutation(createProductMutation);
+  const { viewer } = useAuth();
+
+  const userShopOptions = viewer && viewer.adminUIShops && viewer.adminUIShops.map((shop) => ({
+    value: shop._id,
+    label: shop.name
+  }));
+
+  let preSelectedShop;
+
+  if (userShopOptions && userShopOptions.length === 1) {
+    preSelectedShop = userShopOptions[0];
+  }
```

Comment:
This could work without reassignment:
```suggestion
  const preSelectedShop = userShopOptions && userShopOptions.length === 1 && userShopOptions[0];
```

Reply:
Yup, much better this way and ESLint was complaining about this section anyways. Thanks!